AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -765,8 +766,9 @@ - (void)storeDownloadedReleaseDetails:(nullable MSReleaseDetails *)details {
   [MS_USER_DEFAULTS setObject:groupId forKey:kMSDownloadedDistributionGroupIdKey];
   [MS_USER_DEFAULTS setObject:releaseId forKey:kMSDownloadedReleaseIdKey];
   [MS_USER_DEFAULTS setObject:releaseHashes forKey:kMSDownloadedReleaseHashKey];
-  MSLogDebug([MSDistribute logTag], @"Stored downloaded release hash(es) (%@) and id (%@) for later "
-                                    @"reporting.",
+  MSLogDebug([MSDistribute logTag],
+             @"Stored downloaded release hash(es) (%@) and id (%@) for later "
```

Comment:
Formatter isn't smart enough to deal with this kind of format. It should be at least one less lines.